common/src/models/resource.rs

```diff
@@ -148,6 +153,9 @@ fn get_persist_table(persist_instances: &[&Response], service_name: &str) -> Str
             .add_attribute(Attribute::Bold)]);
 
     for _ in persist_instances {
+        // TODO: add some information that would make the persist resource identifiable.
+        // This requires changing the backend resource list response to include a resource identifier
+        // that can be used to query for more info related to a resource.
```

Comment:
I think we can leave the persistence as is for now... unless if there was a request to change :smile: